### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by ignatius22**
This is Using Bootstrap project of the Newsweek homepage, there is a full description of the project in the README file, I was unsure how in-depth the Newsweek should go I understood to add some content to the site. I decided to submit the project now for the review.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1729 @@
...
+          <li class="nav-item border-right">
+            <a class="nav-link" href="#">The Debate</a>
+          </li>
+          <li class="nav-item border-right">
+            <a class="nav-link" href="#">Vintage</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Weather</a>
+          </li>
+          <li class="input-search">
```
Comment (lucasmazo32) [inline index.html]: - [x] This section breaks for large screens. Kindly fix that.

Source | Clone
-- | --
image | image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1729 @@
...
+                <a href="#"
+                  >Protest Planned in Minneapolis Against Police Brutality After
+                  Death of Man Kneeled on By Cops</a
+                >
+              </li>
+            </ul>
+          </div>
+          <div class="group--2 col-lg col-md-6 col-sm-12 row m-auto">
+            <div class="d-flex flex-column p-3 card-class">
+              <div class="block-title">TOP STORY</div>
```
Comment (lucasmazo32) [inline index.html]: - [x] The top story should go first on small screens.

Source | Clone
-- | --
image | image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1729 @@
...
+          </div>
+        </div>
+      </div>
+
+      <div class="magazine container-fluid">
+        <div
+          class="magazine-title mb-2 p-3 border-bottom d-flex align-items-center justify-content-center"
+        >
+          <span class="icon"></span>
+          IN THE MAGAZINE
```
Comment (lucasmazo32) [inline index.html:784]: - [x] There should be four columns for medium devices in this section.

Source | Clone
-- | --
image | image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,1729 @@
...
+              </div>
+            </div>
+          </div>
+
+          <div class="editors container-fluid">
+            <div
+              class="magazine-title mb-2 p-3 border-bottom d-flex align-items-center justify-content-center"
+            >
+              <span class="icon"></span>
+              EDITOR'S PICK
```
Comment (lucasmazo32) [inline index.html:938]: - [x] Editor's pick should have two columns for medium devices.

Source | Clone
-- | --
image | image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,1729 @@
...
+                    </article>
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div class="section-pick-1 border-top p-3 container-fluid">
+              <div class="container-fluid">
+                <h1 class="block-title--1 text-capitalize">
+                  <a href="#">World</a>
```
Comment (lucasmazo32) [inline index.html:1182]: - [x] Kindly check this section for medium devices.

Source | Clone
-- | --
image | image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,1729 @@
...
+                    </article>
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div class="subscribe-book mt-3 mb-4">
+              <div class="row container-fluid">
+                <div class="col-md-7 col-lg-8 col-sm-12 col-xs-12">
+                  <div class="block-title1 red">Subscribe</div>
```
Comment (lucasmazo32) [inline index.html:1594]: - [x] This section should have two columns for medium devices. Kindly check that.

Source | Clone
-- | -- 
image | image

- [ ] Kindly check the layout for small devices.

Source | Clone
-- | --
image | image

**Review by lucasmazo32 — CHANGES_REQUESTED** [review]
# Changes required :wrench:

You are doing a great job so far, and the project is looking good. 

For the project to get approved, kindly implement the changes mentioned below.

### Happy coding! :computer:

## Lucas Mazo

File: index.html [hunk 7]
```diff
@@ -0,0 +1,1752 @@
...
+                  <div
+                    class="gray p-5 row justify-content-around align-items-center"
+                  >
+                    <img
+                      class="col-lg-4 col-md-4 col-sm-12"
+                      src="resources/may-29-june-06-2020-cover.jpg"
+                      alt=""
+                    />
+                    <div class="info col-lg-8 col-sm-12 col-md-8">
+                      <h3 class="subscribe-block">
```
Comment (lucasmazo32) [inline index.html]: - [x] Kindly add the propper layout for these images like on the source page. Dealing with absolute and relative positioning is a great tool that will help you in the future. You can also check the source page to see how they achieve it.

Source | Clone
-- | --
image | image

**Review by lucasmazo32 — CHANGES_REQUESTED** [review]
# Changes required :wrench:

You are doing a great job so far, the project is looking good and you did a great job implementing the changes.

For the project to get approved, kindly implement the changes mentioned below.

### Happy coding! :computer:

## Lucas Mazo

**Comment by santiago-dev96** [comment]
# Pull Request under Review by @santiago-rodrig (TSE)

**Review by santiago-dev96 — CHANGES_REQUESTED** [review]
# Changes required 🚫

This project is almost ready for approval, but I think you haven't fully understood how to use CSS positioning.

1. Try to make the subscribe section more similar to the original page

**Original**

Screen Shot 2020-06-05 at 9 15 31 AM

**Yours**

Screen Shot 2020-06-05 at 9 15 02 AM

I recommend you to take a read to [this](https://developer.mozilla.org/en-US/docs/Web/CSS/position).

After implementing the changes submit another CRR.

You can find me in slack as **srodrig**, happy coding! :wave:

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

**Review by andresporras3423 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:
**Hello @ignatius22** 
Amazing job :clap: :clap: you have met all the requirements for this project
Now you are ready to continue with your next challenge :muscle: 
Don't forget to merge with the master branch

Keep going like this,

Oscar Andrés Russi Porras

**Pull request merged**